juan-tex-edinburgh.md

```diff
@@ -0,0 +1,16 @@
+# Edinburgh, UK
+
+### Resturants
+
+- Piatto Verde
+- Domenicos
```

Comment:
Hahaha, I don't know. They've just been recommended to me by a food critic, so I'm looking forward to trying them :D